routing/payment_session.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+// BlockPadding is used to increment the finalCltvDelta value for the last hop
+// to prevent an HTLC being failed if a block is mined while it's in-flight.
+var BlockPadding uint16 = 1
```

Comment:
make `const`? also discussed with @Roasbeef and decided maybe bump to 3 to be conservative